src/components/core/PivotTable.tsx

```diff
@@ -967,6 +967,29 @@ export class PivotTableInner extends BaseVisualization<IPivotTableInnerProps, IP
         }
     };
 
+    private getInfiniteInitialRowCountRowCount() {
+        // for init load this method return page after sort change return last execution rowcount
```

Comment:
reduce this comment to the explanation why this is done - scrollbar removal. What is does is readable from code

Reply:
Done